modules/jaggery-apps/portal/conf/site.json

```diff
@@ -1,10 +1,8 @@
 {
     "proxy" : {
-        "proxyHost" : "",
-        "proxyHTTPSPort" : "",
-        "proxyContextPath" : ""
+        "serviceOrigin" : "${carbon.protocol}://${carbon.host}:${carbon.management.port}"
```

Comment:
The 'serviceOrigin' does not match with 'proxy' related configs. Shall we introduce a new section like 'service' or 'server'.